flocker/provision/_ssh/_conch.py

```diff
@@ -20,10 +20,13 @@
     _NewConnectionHelper,
     # https://twistedmatrix.com/trac/ticket/7862
     _ReadFile, ConsoleUI,
```

Comment:
:question: I was wondering if there's a public API that we could use to request a PTY.

I read:
 *  http://as.ynchrono.us/2011/08/twisted-conch-in-60-seconds-protocols.html


which talks about overriding `request_pty_req` on the session. Is that an option here?